<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
border-image:url(:/icon/icon/timg.jpg);
}
#MainWindow *{
border-image:url();
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <widget class="QTabWidget" name="storeTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>760</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="storePage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>商城</string>
     </attribute>
     <widget class="QTableView" name="GoodsTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>740</width>
        <height>371</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addToCartButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>添加至购物车</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeShowComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <item>
       <property name="text">
        <string>所有品类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BOOK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CLOTH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FOOD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PHONE</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="searchByNameLineEdit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,135,0);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sortComboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <item>
       <property name="text">
        <string>默认排序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>价格从低至高</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>价格从高至低</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cartPage">
     <attribute name="title">
      <string>我的购物车</string>
     </attribute>
     <widget class="QTableView" name="CartTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>740</width>
        <height>351</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>提交订单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>420</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifyQuantityButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>420</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>修改数量</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>140</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>折后：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="priceSumLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 135, 0);</string>
         </property>
         <property name="text">
          <string>XXXXXXXXXXX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>元</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>140</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>满减：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fullThenReduceLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 135, 0);</string>
         </property>
         <property name="text">
          <string>XXXXXXXXXXX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>元</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>370</y>
        <width>163</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>实际应付：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="paySumLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 135, 0);</string>
         </property>
         <property name="text">
          <string>XXXXXXXXXXX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>元</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>151</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>总原价：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="oriPriceSumLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 135, 0);</string>
         </property>
         <property name="text">
          <string>XXXXXXXXXXX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>元</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="userInfoPage">
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <attribute name="title">
      <string>我的信息</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 135, 0);</string>
      </property>
      <property name="text">
       <string>我的银行卡：</string>
      </property>
     </widget>
     <widget class="QTableView" name="bankAccountTableView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>320</width>
        <height>180</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="accountMagButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>200</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>添加银行卡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifyUserInfoButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 135, 0);</string>
           </property>
           <property name="text">
            <string>昵称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 135, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 135, 0);</string>
           </property>
           <property name="text">
            <string>手机号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="phoneLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 135, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="disBindingButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>240</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>解绑银行卡</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>331</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/icon/icon/kobe.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>label_3</zorder>
     <zorder>bankAccountTableView</zorder>
     <zorder>accountMagButton</zorder>
     <zorder>modifyUserInfoButton</zorder>
     <zorder>layoutWidget1</zorder>
     <zorder>disBindingButton</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="manageEntryButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>管理员入口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginEntryButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerEntryButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
